app/models/user.rb

```diff
@@ -12,6 +12,10 @@ class User < ApplicationRecord
   has_many :notifications, foreign_key: :receiver_id, dependent: :destroy, inverse_of: :sender
   has_many :user_notification_timings, dependent: :destroy
   has_many :notification_timings, through: :user_notification_timings
+  has_many :relationships, class_name: 'Relationship', foreign_key: :follower_id, dependent: :destroy
+  has_many :reverse_of_relationships, class_name: 'Relationship', foreign_key: :followed_id, dependent: :destroy
+  has_many :followings, through: :relationships, source: :followed
+  has_many :followeres, through: :reverse_of_relationships, source: :follower
```

Comment:
```suggestion
  has_many :followers, through: :reverse_of_relationships, source: :follower
```
ではないでしょうか? 全体置換で直してくださると 👍

Reply:
ご指摘ありがとうございます。
修正しました。

Reply:
[090bca5](https://github.com/sh0327akg/mokumoku/pull/2/commits/090bca5b89aadcf75312d6a3528aa74372a1ea77)←こちらのコミットでfollowersに修正しています。